src/Parser/BitcoinMoneyParser.php

```diff
@@ -39,8 +33,12 @@ public function __construct(MoneyParser $innerParser, $fractionDigits)
      */
     public function parse($money, $forceCurrency = null)
     {
+        if (!is_string($money)) {
+            throw new ParserException('Money should be a string');
```

Comment:
Hm, maybe we could improve this text a little. Money usually refers to the object or the amount. Maybe unformatted/raw money should be string?